tensorflow/core/api_def/base_api/api_def_Expm1.pbtxt

```diff
@@ -2,6 +2,18 @@ op {
   graph_op_name: "Expm1"
   summary: "Computes exponential of x - 1 element-wise."
```

Comment:
Actually, can you change this summary? I think it is ambiguous at best. 

Maybe better: "Computes `exp(x)` - 1 element-wise."